baybe/objectives/desirability.py

```diff
@@ -123,6 +110,16 @@ def _validate_weights(self, _, weights) -> None:  # noqa: DOC101, DOC103
                 f"Specified number of targets: {lt}. Specified number of weights: {lw}."
             )
 
+    @property
+    def targets(self) -> tuple[Target, ...]:  # noqa: D102
```

Comment:
If this property just returns the private `_targets`, what is the point of making that field private?

Reply:
I know it seems weird at first glance but it's actually an elegant (I think) solution to a problem related to subclasses not sharing the same attributes. Don't want to type many sentences here to explain so:
* If you are curious, try to find out yourself why simply using a `targets` attribute would not work
* If you trust me and don't care: resolve
* If you want to now but are lazy: call me 
:D

Reply:
If you tell me that there is a reason behind it I trust you ;) Just wanted to make sure that this is the case as this could also have been an occasion of just overlooking a potential easier solution ;)

Reply:
appreciated